commit 73bc3ea559c7324f47d4283d56a42d1840ec4832
Author: Dave Eddy <dave@daveeddy.com>
Date:   2019-03-12T12:22:20-04:00 (7 months ago)
    
    joyent/boray#6 want `make check`
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..06ff1d7
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,7 @@
+#
+# Copyright 2019 Joyent, Inc
+#
+
+.PHONY: check
+check:
+	cargo check
